src/validators/v1/organization-extensions.js

```diff
@@ -35,6 +42,30 @@ module.exports = {
 			.isIn(allowedResourceTypes)
 			.withMessage(`resource_type is invalid, must be one of: ${allowedResourceTypes.join(', ')}`)
 
+		req.checkBody('data_managers')
+			.optional({ checkFalsy: true })
+			.custom((value) => {
+				// Allow empty array explicitly
+				if (Array.isArray(value) && value.length === 0) {
+					return true
+				}
+
+				// If it's an array, validate each element
+				if (Array.isArray(value)) {
```

Comment:
@adithyadinesh0412  in create api data manager is  a comma seperated string, then why in the update api its accepting as an array also. We can keep the same thing in update also

Reply:
No in create also we are accepting it as array of string only.  ```.matches(/^[A-Za-z]+(?:\s*,\s*[A-Za-z]+)*$/)``` This regex checks if its array of string. @priyanka-TL